### PR TITLE
[OVERFLOW-285] Create API for adding, editing, deleting Members of the team

### PR DESCRIPTION
**Pull request by kenorenzsun**
## Backlog Link
[SUN_OVERFLOW-285](https://framgiaph.backlog.com/view/SUN_OVERFLOW-285)
## Commands
None
## Pre-conditions
None
## Expected Output
Should be able to 
- [ ] add members of team
- [ ] update member of team
- [ ] delete member of team
## Notes
None
## Screenshots
image
image
image
image

### CODE REVIEW

File: backend/app/GraphQL/Mutations/DeleteMember.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+     * @param  null  $_
+     * @param  array{}  $args
+     */
+    public function __invoke($_, array $args)
+    {
+        $authId = Auth::id();
+        $member = Member::find($args['id']);
+
+        if ($authId != $member->team->teamLeader->id) {
+            throw new CustomException('You are not allowed to add members to this team');
```
Comment (IanUrriza10) [inline backend/app/GraphQL/Mutations/DeleteMember.php]:
```suggestion
            throw new CustomException('You are not allowed to remove members from this team');
```

File: backend/app/GraphQL/Mutations/UpdateMember.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+     * @param  array{}  $args
+     */
+    public function __invoke($_, array $args)
+    {
+        $authId = Auth::id();
+        $member = Member::find($args['id']);
+        $updateData = Arr::except($args, ['tags']);
+
+        if ($authId != $member->team->teamLeader->id) {
+            throw new CustomException('You are not allowed to add members to this team');
```
Comment (IanUrriza10) [inline backend/app/GraphQL/Mutations/UpdateMember.php]:
```suggestion
            throw new CustomException('You are not allowed to edit members in this team');
```

**Comment by IanUrriza10** [comment]
Review: 15 mins
Comments 2
Iteration: 1

**Comment by presyos** [comment]
Review: 9 mins
Comments: 0
Iteration: 0

**Pull request merged**